### DISCUSSION TITLE
v2.7.8

### DISCUSSION THREAD

**Discussion by Limraj** [SCADA-LTS/Scada-LTS#2946] [General]
## 

The changelist is listed below:

### 📄️ GitHub issues list overview:

1. #2000 Removal of unnecessary api calls
2. #2030 Improvements and fixed chart
3. #2127 Fixed ConcurrentModificationException in MiscDwr.getDataFromList
4. #2696 Generation of active events by handlers
5. #2708 Fixed 'Narrowly avoided an infinite loop in execute' for HttpSenderRT
6. #2711 Integration with FuscaBR library (#2503, #2668, #2701)
7. #2773 Enhance copy/paste datapoint properties by including event detectors (#1740)
8. #2782 Point locator test in Modbus IP blocked after bad configuration choice
9. #2845 Fixed message validation in event search
10. #2864 Adapting application to work in address without name as ROOT (#2721)
11. #2876 Added XID column for context points in Meta Data Point
12. #2885 Fixed feature Enabled Event Pending Cache
13. #2887 Parametrized pool Medium/High Priority
14. #2896 Fixed alert language in watchlist view
15. #2904 [Feature] Support for ARM64 devices? (#2901)
16. #2907 Synoptic URL not working
17. #2910 Fixed exceeded a secondary rate limit for github api
18. #2912 Created CODEOWNERS file
19. #2915 Fixed Changing register range for coil and input status in Modbus IP
20. #2917 Added Point locator test fields validation in Modbus IP
21. #2922 Fixed not recalculate Meta Data Point if Context is empty
22. #2925 Fixed copying data point in freshly created data source
23. #2926 Fixed icons placement on data point list
24. #2927 Fixed Graphical views page when views empty
25. #2930 Fixed high pool priority and database termination sequence
26. #2944 Fixed create WatchList

## What's Changed
* Fix/#2668 fixed alignment tool making components go out of bounds in graphical view by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2713
* Release/2.7.7.3 by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2881
* #2876 Added XID column for context points in Meta Data Point  by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2883
* #2885 Fixed feature Enabled Event Pending Cache by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2886
* Feature/#2503 add alignment and template feature for html and js by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2884
* #2503 add alignment and template feature for html and js - fixed chan… by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2889
* #2773 enhance copy paste datapoint properties by including event detectors by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2895
* #2127 Fixed ConcurrentModificationException in MiscDwr.getDataFromList by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2890
* #2901 add a docker arm64 image: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2906
* #2907 Bugfix: URL now loads synoptic panel - by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2909
* #2910 Fixed exceeded a secondary rate limit for github api - use GITH… by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2911
* #2912 Created CODEOWNERS file by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2913
* #2782 Point locator test in Modbus IP blocked after bad configuration choice by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2898
* #2915 Fixed Changing register range for coil and input status in Modbus IP by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2916
* #2922 Fixed not recalculate Meta Data Point if Context is empty: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2923
* #2845 Fixed message validation in event search by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2899
* #2701 Fixed apply templates in HTML/Server-side view component: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2914
* #2927 Fixed Graphical views page when views empty: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2928
* #2887 Parametrized pool Medium/High Priority: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2921
* Fix/#2925 fixed copying data point in freshly created data source by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2934
* #2917 Added Point locator test fields validation in Modbus IP by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2919
* #2896 Fixed alert language in watchlist view by @Patrykb0802 in https://github.com/SCADA-LTS/Scada-LTS/pull/2933
* #2932 Fixed French language causing system exception in graphical vie… by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2936
* #2708 Fixed 'Narrowly avoided an infinite loop in execute' for HttpSe… by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2937
* #2926 Fixed icons placement on data point list2: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2941
* #2696 Generation of active events by handlers: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2935
* #2864 Adapting application to work in address without name as ROOT: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2929
* Fix/#2930 fixed high pool priority and database termination sequence by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2931
* #2944 Fixed create WatchList: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2945
* #2000 Removal of unnecessary api calls: by @Limraj in https://github.com/SCADA-LTS/Scada-LTS/pull/2943

**Full Changelog**: https://github.com/SCADA-LTS/Scada-LTS/compare/v2.7.7.3...v2.7.8

MD5 (Scada-LTS.war) = ``

<hr /><em>This discussion was created from the release <a href='https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.8'>v2.7.8</a>.</em>

**Comment by fabiodurao** [comment]
I installed this version (2.7.8) on Ubuntu 24.04.1 LTS Server with version 1.4.3 of the Linux installer and during the installation of mysql_start.sh I got an error saying that libaio.so.1 could not be found. I tried to install it as I have done in other installations where I had this problem, but it is no longer available through the "apt install" repository. The option that exists is libaio.so.1l64. I installed it, but the installer still did not find the file because it was looking for it with the other name. So I made a link between them and it worked:

ln -s /lib/x86_64-linux-gnu/libaio.so.1t64 /usr/lib/x86_64-linux-gnu/libaio.so.1

I leave the solution as a suggestion for future updates to the installer.

**Reply by fabiodurao** [reply]
I'm returning to this topic because today I deployed a new Scada-LTS version to my Ubuntu 24.04 server and I can no longer install libaio.so.1 or libaio.so.1t64 using sudo apt install. They've been discontinued for security reasons.

Today's solution was to manually download and install the package. I don't need to do the "ln" because I've already installed the version requested by MySQL:

```
wget http://archive.ubuntu.com/ubuntu/pool/main/liba/libaio/libaio1_0.3.112-13build1_amd64.deb
sudo apt install ./libaio1_0.3.112-13build1_amd64.deb
```
